CounterpointGenerator/CounterpointGenerator/Note.cs

```diff
@@ -5,9 +5,10 @@ public class Note
     {
         int NoteUnit { get; set; }
 
-        /*public Note()
+        public Note(string item)
         {
+            NoteUnit = Int32.Parse(item);
```

Comment:
Oh, I see what's causing the merge conflict. I had gone and updated Note so instead of `NoteUnit` it uses `Pitch` then added a field for Length, then added constructors for `Note(int Pitch)` and `Note(int Pitch, int Length)` and an `Equals(Note n)` method.

IOTranslator is still using NoteUnit and added a constructor for a string input. Since all the string constructor does is directly parse the parameter to an int I'd like to suggest moving that step outside the Note class (so the input translator would parse, I believe) and just leaving the int constructor, but I'm not opposed to leaving a string constructor if that would be useful to you later down the line.

The int constructors do need to stay though as I use them heavily.

Reply:
To get this merged though it's a simple enough fix that even the github web editor is available to do it. I'm not gonna jump the gun and do it myself so this conversation can happen, but if you want to just merge as-is and update later:

Delete the 
```csharp
<<<<<<< IOTranslator
```
```csharp
=======
```
and
```csharp
>>>>>>> main
```
lines

And replace
```csharp
public Note(string item)
{
    NoteUnit = Int32.Parse(item);
}
```
with
```csharp
public Note(string item)
{
    Pitch = Int32.Parse(item);
}
```

But I still do think that the int parsing should be taken out of Note and moved to the input translator.

Reply:
exactly, the "NoteUnit" is meant to be the "Pitch". So, I deleted the "NoteUnit" member variable and use "Pitch" instead in the string constructor

Reply:
but I currently prefer the parsing to be in the Note because Input in InputTranslator accepts MelodyLine which consists of Note, it seems more intuitive to parse in the smallest unit, Note class.